README.md

```diff
@@ -81,6 +96,20 @@ var allAccounts = await gdaxClient.AccountsService.GetAllAccountsAsync();
 ###### User Account ######
 - GetTrailingVolumeAsync() - get 30-day trailing volume for all products
 
+###### Websocket Feed ######
```

Comment:
not a huge deal but i think this should also have its own section. doesn't really belong with the services